### ISSUE TITLE
Wrong sortable_update url with multiple sortable models

### ISSUE THREAD

**Issue by beslave** [jrief/django-admin-sortable2#79]
I think there is a bug.
```{% url 'admin:sortable_update' %}``` always return 'sortable_update' url for last registered url SortableAdmin.
For example, I have three models: ModelOne, ModelTwo, ModelThree.
All this model are registered as SortableAdmin instances for admin panel.
I go to /admin/example_app/model_three and see that

```
var sortable_current_page = '/admin/example_app/model_three/adminsortable2_update';
```
It's fine, but when I go to modelOne or modelTwo page, the link is the same. So if I drag some ModelOne/ModelTwo the order field will be update for ModelThree instances (or error if instance with same order field value is absent)

![different_urls](https://cloud.githubusercontent.com/assets/1387623/11743092/6784c8de-a00d-11e5-8a0a-ee2cb914e59e.png)
As you see url for sort_update is wrong (it's url for some Section model, but page is for Contacts list)

![debug_info](https://cloud.githubusercontent.com/assets/1387623/11743091/6784273a-a00d-11e5-91d6-c28190ad43f2.png)
It's error for case when I try to sort some instance of MyModel with order=X but OtherModel.objects.get(order=X) is absent

**Comment by teddybeermaniac** [comment]
Hi, I'm the one who broke this with my pull request some time ago, so it's on me to fix it. ;)
Could you please check my [fork's development branch](https://github.com/teddybeermaniac/Django-Admin-Sortable-2/tree/development) and see if it works? On my side it does, but I'll sleep better if my next PR won't break anything even more. :)

**Comment by beslave** [comment]
Hi. Yes, it works as expected. Different models - different urls.
Thank you for quick solution:)

**Comment by jrief** [comment]
fixed via #80
